### PR TITLE
fr_FR.lang (1.13.2)

### PR DESCRIPTION
**Pull request by Neerwan**

### CODE REVIEW

File: src/main/resources/assets/waila/lang/en_us.json [hunk 1]
```diff
@@ -1,80 +1,80 @@
...
 
   "config.waila.plugin_waila": "Waila",
-  "config.waila.plugin_waila.show_registry": "Show Registry Names",
-  "config.waila.plugin_waila.show_entities": "Show Entities",
-  "config.waila.plugin_waila.show_entity_hp": "Show Entity Health",
-  "config.waila.plugin_waila.show_states": "Show States",
+  "config.waila.plugin_waila.show_registry": "Afficher les noms du registre",
+  "config.waila.plugin_waila.show_entities": "Afficher les entités",
+  "config.waila.plugin_waila.show_entity_hp": "Afficher la vie des entités",
+  "config.waila.plugin_waila.show_states": "Afficherles états",
```
Comment (Pyrofab) [inline src/main/resources/assets/waila/lang/en_us.json:77]: Missing space

File: src/main/resources/assets/waila/lang/en_us.json [hunk 2]
```diff
@@ -1,80 +1,80 @@
...
+  "config.waila.plugin_minecraft.redstone": "Niveau de puissance",
+  "config.waila.plugin_minecraft.jukebox": "Disque",
+  "config.waila.plugin_minecraft.villager_profession": "Profession de villageois",
 
   "config.waila.plugin_waila": "Waila",
-  "config.waila.plugin_waila.show_registry": "Show Registry Names",
-  "config.waila.plugin_waila.show_entities": "Show Entities",
-  "config.waila.plugin_waila.show_entity_hp": "Show Entity Health",
-  "config.waila.plugin_waila.show_states": "Show States",
+  "config.waila.plugin_waila.show_registry": "Afficher les noms du registre",
```
Comment (Pyrofab) [inline src/main/resources/assets/waila/lang/en_us.json:74]: "noms **de** registre"

File: src/main/resources/assets/waila/lang/en_us.json [hunk 3]
```diff
@@ -1,80 +1,80 @@
...
-  "config.waila.format_block_name": "Block Name",
-  "config.waila.format_fluid_name": "Fluid Name",
-  "config.waila.format_entity_name": "Entity Name",
-  "config.waila.format_registry_name": "Registry Name",
+  "config.waila.formatting": "Mise en forme",
+  "config.waila.format_mod_name": "Nom du mod",
+  "config.waila.format_block_name": "Nom du bloc",
+  "config.waila.format_fluid_name": "Nom du fluide",
+  "config.waila.format_entity_name": "Nom de l'entité",
+  "config.waila.format_registry_name": "Nom du registre",
```
Comment (Pyrofab) [inline src/main/resources/assets/waila/lang/en_us.json:59]: This is the entry's registry name as in the name in the registry. Just like your family name is not "nom de la famille", this should be "Nom **de** registre"

File: src/main/resources/assets/waila/lang/en_us.json [hunk 4]
```diff
@@ -1,80 +1,80 @@
...
-  "config.waila.hide_from_players_desc": "Hide tooltip while the player list is open",
-  "config.waila.hide_from_debug": "Hide from Debug",
-  "config.waila.hide_from_debug_desc": "Hide the tooltip while the debug menu is open",
-  "config.waila.display_item": "Display Item",
-  "config.waila.display_item_desc": "Display the item gotten from the block",
-  "config.waila.tts": "Enable TTS",
-  "config.waila.tts_desc": "Read out block and entity names with the system's text to speech processor",
+  "config.waila.general": "Général",
+  "config.waila.display_tooltip": "Afficher l'infobulle",
+  "config.waila.display_tooltip_desc": "Récupérer des données et rendre l'infobulle",
```
Comment (Pyrofab) [inline src/main/resources/assets/waila/lang/en_us.json:27]: "Afficher l'infobulle". Literal translation of "render tooltip" is confusing.

File: src/main/resources/assets/waila/lang/en_us.json [hunk 5]
```diff
@@ -1,80 +1,80 @@
...
-  "config.waila.plugin_minecraft.repeater": "Repeater Delay",
-  "config.waila.plugin_minecraft.comparator": "Comparator Mode",
-  "config.waila.plugin_minecraft.redstone": "Power Level",
-  "config.waila.plugin_minecraft.jukebox": "Active Record",
-  "config.waila.plugin_minecraft.villager_profession": "Villager Profession",
+  "config.waila.plugin_minecraft.display_furnace_contents": "Contenu des fourneaux",
+  "config.waila.plugin_minecraft.hide_infestations": "Cacher les blocs infestés",
+  "config.waila.plugin_minecraft.spawner_type": "Type de générateur (spawner)",
+  "config.waila.plugin_minecraft.crop_progress": "Progression des plantes",
+  "config.waila.plugin_minecraft.lever": "Statut de levier",
```
Comment (Pyrofab) [inline src/main/resources/assets/waila/lang/en_us.json:66]: "État du levier". "Statut" is not exactly wrong but weird

**Review by Pyrofab — CHANGES_REQUESTED** [review]
Disregarding the fact that you are committing changes to the English localization, there are a couple issues

**Comment by Neerwan** [comment]
Hi,

Oopsy, I'm going to fix the issues you are pointing out soon.

Thanks for you feedback.

**Pull request closed without merging**